README.md

```diff
@@ -91,22 +92,39 @@ This release changes the way in which location data is collected to better align
 - [improvement] [cht-android#111](https://github.com/medic/cht-android/issues/111): Compliance with Play Store developer policies for PII collection disclosure
 - [bug] [cht-core#6648](https://github.com/medic/cht-core/issues/6648): Blank screen when launching external apps from CHT Android app
 
+
 # Development
 
-1. Install Android SDK
-2. Clone the repo
-3. Plug in your phone. Check it's detected with `adb devices`
-4. Execute: `make` (will also push app unto phone)
+1. Install the [Android SDK](https://developer.android.com/studio#command-tools) and Java 8+ (it works with the OpenJDK versions).
+2. Clone the repo.
+3. Plug in your phone. Check it's detected with `adb devices`.
+4. Execute: `make` (will also push app into phone).
+
+To only **build and assemble** the APKs for certain flavor and rendering engine, execute `./gradlew assemble[Flavor][Engine]Debug`, e.g. to assemble the unbranded flavor for the webview rendering engine: `./gradlew assembleUnbrandedWebviewDebug`. For the XWalk engine: `./gradlew assembleUnbrandedXWalkDebug`.
+
+The APKs are stored in the folder `build/outputs/apk/[Flavor][Engine]/debug/`, in the example above: `build/outputs/apk/unbrandedWebview/debug/`.
+
+To only execute the **linter checks**, execute: `./gradlew androidCheckstyle`.
+
+The [Android Studio](https://developer.android.com/studio) can be used to build and launch the app instead. Be sure to select the right flavor and rendering engine from the _Build Variants_ dialog (see [Build and run your app](https://developer.android.com/studio/run)). To launch the app in an emulator, you need to uncomment the code that has the strings for the `x86` or the `x86_64` architecture in the `build.gradle` file, in the `android` / `splits` / `include` section.
```

Comment:
I made it a sub section, but the links provided should be enough for the user to go deeper of how to use Android Studio and how to setup emulators. I only wanted to mention about the selection of the variant and the commented emulators' arch because are the typical question that I had and others had in the past when starting to work with the app.